src/main/logger.ts

```diff
@@ -1,20 +1,30 @@
+import { app } from "electron";
 import winston from "winston"
 import { isDebugging } from "../common/vars";
 
-const options = {
-  colorize: true,
+const consoleOptions: winston.transports.ConsoleTransportOptions = {
   handleExceptions: false,
-  json: false,
   level: isDebugging ? "debug" : "info",
 }
 
+const fileOptions: winston.transports.FileTransportOptions = {
+  handleExceptions: false,
+  level: isDebugging ? "debug" : "info",
+  filename: "lens.log",
+  dirname: app.getPath("logs"),
```

Comment:
In my dev env, this line is problematic and `app === undefined`. Is it complaining the same for you?

Reply:
Going back again, yes it is. Though I think that it is more of an associated problem. Since it seems like `main/logger` shouldn't be important by anything in `common` or `renderer` since that leaks abstraction.

Reply:
I have decided that the best course of action for this PR is to just do `|| remote.app` since it is the smallest fix. However, I really think that this should be fixed properly. Namely, the following rules:

1. Nothing in `common` should import anything from either `renderer` or `main`
2. Nothing in `main` should import anything from `renderer`
3. Nothing in `renderer` should import anything from `main`.